lisa_commands/request_helper.py

```diff
@@ -48,20 +48,24 @@ def submit_slack_request(data, chat_action):
     # Fire off the request!
     return urllib.request.urlopen(request, json.dumps(data).encode('utf-8')).read()
 
-def anime_news_network_search(command, query):
+def anime_news_network_search(media_type, query):
     """
     search anilist for series info
+    :type [anime|manga] string
+    :query what to search for
     """
     params = urllib.parse.urlencode({
         'id':155,
-        'search': query,
-        'type': command
+        'search': query.strip(),
+        'type': media_type
     })
 
     data = urllib.request.urlopen(r'https://www.animenewsnetwork.com/encyclopedia/reports.xml?' + params).read().decode('utf-8')
+    logging.warning(data)
```

Comment:
What logging do you have setup? Is this just standard debug logging or are you saving the results?

Reply:
This line should probably be removed. `logging.warning` is the only way I knew to display debug information in the AWS console so I can examine what's going on. Ideally this will change as we add unit and integration testing.